app/views/rubygems/show.html.erb

```diff
@@ -115,6 +115,17 @@
         </i>
       </h2>
 
+      <h2 class="gem__ruby-version__heading t-list__heading">
+        <%= t '.native_extensions' %>:
+        <i class="gem__ruby-version">
+          <% if !@latest_version.extensions.nil? %>
```

Comment:
I thought about this firstly, but because of tri-state it's not reasonable. If it's not there I can't tell for sure: is there NO dependencies or we just don't know about them...